spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -2002,13 +2002,17 @@ def __mark_occurrences(self):
         cursor = self.__find_first(text)
         self.occurrences = []
         extra_selections = self.get_extra_selections('occurrences')
-
+        first_ocurrence = None

```

Comment:
Please change `first_ocurrence` to `first_occurrence` (the first one is mispelled).